src/mol-plugin-state/formats/volume.ts

```diff
@@ -197,15 +198,17 @@ export const DscifProvider = DataFormatProvider({
     parse: async (plugin, data, params?: DsCifParams) => {
         const cifCell = await plugin.build().to(data).apply(StateTransforms.Data.ParseCif).commit();
         const b = plugin.build().to(cifCell);
-        const blocks = cifCell.obj!.data.blocks.slice(1); // zero block contains query meta-data
+        const blocks = cifCell.obj!.data.blocks.slice(0); // iterate over all blocks as even 0th can contain data
```

Comment:
```suggestion
        const blocks = cifCell.obj!.data.blocks;
```

No need to call the slice since we are iterating all of them.